test/rabbitmanager_test.js

```diff
@@ -159,10 +159,11 @@ suite('Rabbit Wrapper', function() {
 
     assert(queues instanceof Array);
     assert(queues.length > 0);
-    assert(_.has(queues[0], 'memory'));
-    assert(_.has(queues[0], 'messages'));
-    assert(_.has(queues[0], 'messages_details'));
-    assert(_.has(queues[0], 'messages_ready'));
+    // TODO: the rabbit server doesn't return these keys, it seems
```

Comment:
Awesome, thanks for that context -- indeed, that does seem like it could be problematic..